migrations/migrations/20180326222029-users.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    queryInterface.addColumn('users', 'role', Sequelize.STRING);
+  },
+
+  down: (queryInterface, Sequelize) => {
+    queryInterface.removeColumn('users', 'role', Sequelize.STRING);
```

Comment:
there may be some problems if you dont drop also the type created with the enum for roles which im pretty sure (though not 100%) this will not drop it.